forward_script/src/QueryBuilder.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+
+namespace Wikimedia\ForwardScript;
+
+class QueryBuilder {
+
+	public function getQuery( $info, $pageName, $id = '', $coordinates = [] ) {
+		$category = preg_replace( '/^(:?Category|Kategorie):/', '', $info->category );
```

Comment:
I'm always using non-capturing groups if no capture is needed because they are a little bit faster. I know it's a micro-optimization vs readability tradeoff ...